src/engine/blocks.js

```diff
@@ -554,6 +555,13 @@ class Blocks {
         // A new block was actually added to the block container,
         // emit a project changed event
         this.emitProjectChanged();
+
+        const scratchBlock = ScratchBlocks.getMainWorkspace().getBlockById(block.id);
```

Comment:
Funny, I'm getting an error due to this line?

Reply:
<img width="1213" alt="Screenshot 2024-06-30 at 11 57 04 PM" src="https://github.com/mitmedialab/prg-raise-playground-scratch-vm/assets/95306112/e23041a7-d463-45a7-98d2-b9a07465bb15">